### ISSUE TITLE
Test titles are mixed up in parallel

### ISSUE THREAD

**Issue by ohcibi** [trentmwillis/ember-exam#86]
I'm running

    ember exam --split=4 --parallel --random

I have an acceptance test that fails. The test has a title `visiting /someurl`. In the output of `ember exam` there is a line

    not ok 232 PhantomJS 2.1 - Exam Partition #3 - Unit | Mixin | some mixin: visiting /someurl

The test-module with the title `Unit | Mixin | some mixin` has no failing tests and it neither contains a test with `test("visiting /someurl....`, so it seems that ember exam is mixing up test module titles and tests. When running without `--random` there is no mixup.

**Comment by trentmwillis** [comment]
Ember Exam simply uses [`QUnit.config.random`](https://api.qunitjs.com/config/QUnit.config#qunitconfigseed-string--default-undefined) when using the `--random` option. So it seems likely that this is an issue with QUnit itself and not Ember Exam. That said, a reproduction would be greatly helpful here as this feature has been thoroughly tested.

**Comment by scottkidder** [comment]
The cause of this is being discussed on #81

**Comment by trentmwillis** [comment]
Based on the comment above, I'm going to close this issue as a duplicate of #81. If, in the course of fixing #81, this issue is not resolved, feel free to reopen with a reproduction.
